tools/make/tests.toml

```diff
@@ -9,7 +9,7 @@ description = "Build all permutations of all features for the provided chunk ind
 category = "ICU4X Development"
 install_crate = { crate_name = "cargo-all-features", binary = "cargo-check-all-features", test_arg = ["--help"] }
 install_crate_args = ["--version", "^1.10"]
-env = { RUSTFLAGS = "-Dwarnings" }
+env = { RUSTFLAGS = "-Dwarnings",  "ICU4X_DATA_DIR" = "${CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY}/provider/datagen/tests/data/baked" }
```

Comment:
Is this going to work on Windows? Is it covered by the Windows CI, or alternatively is there some command I can try running to check if it does?